fmriprep/utils/transforms.py

```diff
@@ -38,16 +37,6 @@ def load_transforms(xfm_paths: list[Path], inverse: list[bool]) -> nt.base.Trans
     return chain
 
 
-FIXED_PARAMS = np.array([
-    193.0, 229.0, 193.0,  # Size
-    96.0, 132.0, -78.0,   # Origin
-    1.0, 1.0, 1.0,        # Spacing
```

Comment:
We now use size (`[:3]`), origin (`[3:6]`) and directions (`[9:15]`). We don't deal with the case where Spacing (`[6:9]`) is not 1x1x1, and we are desperately hoping that the directions will never be anything but orthonormal LPS.

I think we probably do want to at least keep the check that `fixed_params[6:]` isn't something we don't know exactly what to do with. (What's currently a `ValueError`.)